tests/test_rest.py

```diff
@@ -1267,24 +1268,30 @@ def patch_group(json: Dict, expect_failure: bool = no_groups) -> Optional[Dict]:
     assert patched_group is not None
     return patched_group
 
+  # if a group has no zones, voluem control won't work as expected
```

Comment:
```suggestion
  # if a group has no zones, volume control won't work as expected
```
Minor spelling error in comment, not a real issue but might as well be fixed